lib/runners/firecracker.nix

```
@@ -86,15 +86,15 @@ in {
     then throw "hotpluggedMem not implemented for Firecracker"
     else if credentialFiles != {}
     then throw "credentialFiles are not implemented for Firecracker"
-    else lib.escapeShellArgs [
+    else lib.escapeShellArgs ([
       "${pkgs.firecracker}/bin/firecracker"
       "--config-file" configFile
       "--api-sock" (
         if socket != null
         then socket
         else throw "Firecracker must be configured with an API socket (option microvm.socket)!"
       )
-    ];
+    ] ++ lib.optional microvmConfig.firecracker.enablePci "--enable-pci");
```

Does it make a lot of sense to have an option to enable PCI? How about doing just:

```suggestion
    ] ++ lib.optional (lib.versionAtLeast pkgs.firecracker.version "1.13.0") "--enable-pci");
```